db/import.go

```diff
@@ -78,9 +78,18 @@ func (db *Database) ImportDoc(docid string, existingDoc *Document, isDelete bool
 
 	// TODO: We need to remarshal the existing doc into bytes.  Less performance overhead than the previous bucket op to get the value in WriteUpdateWithXattr,
 	//       but should refactor import processing to support using the already-unmarshalled doc.
-	rawValue, rawXattr, err := existingDoc.MarshalWithXattr()
-	if err != nil {
-		return nil, err
+
+	var rawValue []byte
+	var rawXattr []byte
```

Comment:
Initializing existingBucketDoc up here with Cas and expiry and then setting existingBucketDoc.Body and Xattr inside the if/else would remove the need for temporary variables.